website/docs/installation/configuration.mdx

```diff
@@ -296,23 +295,23 @@ Starting with authentik 2023.2, multiple modes can be set, and authentik will fa
 Requires authentik 2021.12.5
 :::
 
-Enable the ability for users to change their name, defaults to `true`.
+Enable the ability for users to change their names, defaults to `true`.
 
 ### `AUTHENTIK_DEFAULT_USER_CHANGE_EMAIL`
 
 :::info
 Requires authentik 2021.12.1
 :::
 
-Enable the ability for users to change their Email address, defaults to `false`.
+Enable the ability for users to change their email addresses, defaults to `false`.
 
 ### `AUTHENTIK_DEFAULT_USER_CHANGE_USERNAME`
 
 :::info
 Requires authentik 2021.12.1
 :::
 
-Enable the ability for users to change their Usernames, defaults to `false`.
+Enable the ability for users to change their usernames, defaults to `false`.
 
 ### `AUTHENTIK_GDPR_COMPLIANCE`
```

Comment:
@BeryJu I thought we removed all of those as they're now configured on the settings page in the UI?

Reply:
apparently we missed these

Reply:
https://github.com/goauthentik/authentik/pull/8976